temporian/core/data/duration.py

```diff
@@ -103,3 +111,61 @@ def is_a_date(value: any) -> bool:
         if value is date_type:
             return True
     return False
+
+
+def duration_abbreviation(
+    duration: Duration, cutoff: str = "subseconds"

```

Comment:
Would be nice for this `cutoff` to be an enum if it only has a set of possible values